dozer-cli/src/config_helper.rs

```diff
@@ -67,7 +46,38 @@ pub fn combine_config(config_paths: Vec<String>) -> Result<Option<String>, Confi
     }
 }
 
-fn merge_yaml(
+pub fn add_file_content_to_config(
+    combined_yaml: &mut serde_yaml::Value,
+    name: &str,
+    content: String,
+) -> Result<(), ConfigCombineError> {
+    if name.contains(".yml") || name.contains(".yaml") {
+        let yaml: serde_yaml::Value = serde_yaml::from_str(&content)
+            .map_err(|e| ConfigCombineError::ParseYaml(name.to_string(), e))?;
+        merge_yaml(yaml, combined_yaml)?;
+    } else if name.contains(".sql") {
+        combined_yaml
+            .as_mapping_mut()
+            .expect("Should be mapping")
+            .entry(serde_yaml::Value::String("sql".into()))
+            .and_modify(|s| {
+                let query = s.as_str();
+                *s = match query {
```

Comment:
We should return an error here? The value of `sql` should always be string. If user writes something else, we can let them know by returning error instead of silently override?

Reply:
Right. Changed to error result.